src/client/gui/lib/vm_details/vm_details_general.dart

```diff
@@ -172,8 +172,13 @@ class GeneralDetails extends ConsumerWidget {
       height: baseVmStatHeight,
       label: 'CREATED',
       child: CopyableText(
-        DateFormat('yyyy-MM-dd HH:mm:ss')
-            .format(info.instanceInfo.creationTimestamp.toDateTime()),
+        // Show a dash when VM is still launching and timestamp is therefore 0
```

Comment:
The blob below is barely readable. How about using something like:

```dart
extension InstanceDetailsExtensions on InstanceDetails {
  String formattedCreationTime({required bool isLaunching}) {
    final ts = creationTimestamp;
    if (isLaunching && ts.seconds == 0 && ts.nanos == 0) return '-';
    return DateFormat('yyyy-MM-dd HH:mm:ss').format(ts.toDateTime());
  }
}
```

... and then:

```dart
info.instanceInfo.formattedCreationTime(isLaunching: isLaunching)
```